tdrs-frontend/src/components/NoMatch/NoMatch.jsx

```diff
@@ -1,16 +1,68 @@
-import { useLocation } from 'react-router-dom'
-
 /** This component renders a message signaling to the user that this
  * page does not exist.
  */
 export default function NoMatch() {
-  let location = useLocation()
-
   return (
     <div>
-      <h3>
-        No match for <code>{location.pathname}</code>
-      </h3>
+      <div className="usa-section">
+        <div className="grid-container">
+          <div className="grid-row grid-gap">
+            <main className="" id="main-content">
+              <div className="usa-prose">
+                <h1>Page not found</h1>
+
+                <p className="usa-intro">
+                  We’re sorry, we can’t find the page you&#39;re looking for. It
+                  might have been removed, changed its name, or is otherwise
+                  unavailable.
+                </p>
+
+                <p>
+                  If you typed the URL directly, check your spelling and
+                  capitalization. Our URLs look like this:{' '}
+                  <strong>https://www.acf.hhs.gov/about</strong>.
+                </p>
+                <p>
+                  Visit our homepage for helpful tools and resources, or contact
+                  us and we’ll point you in the right direction.
+                </p>
+
+                <div className="margin-y-5">
+                  <ul className="usa-button-group">
+                    <li className="usa-button-group__item">
+                      <a href="/welcome" className="usa-button">
+                        Visit homepage
+                      </a>
+                    </li>
+                    <li className="usa-button-group__item">
+                      <button
```

Comment:
This doesn't do anything, right? Perhaps we could just have it be a mailto: link.

Reply:
Indeed, @ADPennington what email address would you like to see here?

Reply:
> Indeed, @ADPennington what email address would you like to see here?

`tanfdata@acf.hhs.gov` is our resource mailbox. serves as our help desk.

Reply:
Thanks adding this now. There is another question on the phone number listed below.